src/rpc/connection/dispatch.c

```diff
@@ -199,13 +198,14 @@ int handle_run(connection_request_event_info *info)
     return (-1);
   }
 
-  if (!meta->obj[0].data.string.str) {
+  if (!meta->obj[0].data.string.str || meta->obj[0].data.string.length != 16) {
```

Comment:
yeah. the check is ok. i just wondered what we check here. with the macro it's understandable.